### ISSUE TITLE
[8.0.3] Disconnecting every 10 minutes

### ISSUE THREAD

**Issue by MissFoxxie** [StrangeLoopGames/EcoIssues#11468]
While playing Eco 8.0.3 I'm disconnecting every 10 minutes (connection timed out).
After this happens it's hard to get back into the game. I can't join onto the world via the 'My Worlds' list neither the 'Continue' button. When trying to join after a friend trough Steam friendslist the chance on getting back onto the server is at least at 80%.

I had some disconnection problems on the other patches as well but it was actually playable since i got back onto the servers fairly quick. 

Few informations I can offer:

- I noticed that, in a lot of cases, my CPU gets up to 100%.
- The chance on joining a server gets higher while taking down all graphics to very low 
- My game doesn't find any servers in the search tab (a friend has to find them for me so I can try to join after)
- It's very random how long it takes to get back onto a server. Sometimes it's 20 minutes, sometimes it's first try
- Could it be an internet issue? I'm still waiting on higher wifi speed

PC:

AMD FX(tm)-6300 Six-Core-Processor 3,50GHz
8GB RAM
GeForce GTX 950
64 Bit Win 7

All drivers are up to date.

**Comment by ikiris** [comment]
Seeing this on the Tooshi dev hosted server as well. VERY frustrating, and not great visibility for the people watching the stream.

**Comment by Reyalsorik** [comment]
Can confirm, this was an issue for our server as well. After changing read/write timeouts on stats from 1000 to 500. Our server pretty much runs without issues now. I've made a few other adjustments along the way, but another one that many of my users found useful with their local network was changing tickets from 20 to 10. After making these adjustments, the disconnects from 10 minutes, seem to have gone ahead.

I could very well be forgetting something else I tweaked, but my users are seeing great improvements. Especially with 15-20 out of our 300 people on at a time.

**Comment by eliotcougar** [comment]
I see that on the official sea-otter server too...
After the server has been restarted, it's been running okay for a day or two... Then the intermittent "connection problem" issue (when the whole game pauses for 5 seconds every 10 minutes) reappeared... Eventually, yesterday players were kicked from the game every 10 minutes (probably, when the timeout have become too big)...

**Comment by D3nnis3n** [comment]
Should have been fixed in 8.0.5, is a duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/11068

Reopen if it still happens after that hotfix.

**Comment by ky0uraku** [comment]
still happens on 8.0.7:

every 10 minutes 100% CPU load (1600% on 16-cores Xeon) in-game stuck and sometimes packet loss, leading to disconnect.

![chart2 php](https://user-images.githubusercontent.com/48658286/54497737-44da3900-490f-11e9-8f5d-88d63ad6a626.png)
